src/event-stream/event-stream.interfaces.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ApiProperty } from '@nestjs/swagger';
+
+export interface EventStream {
+  id: string;
+  name: string;
+}
+
+export interface EventStreamSubscription {
+  id: string;
+  name: string;
+  stream: string;
+}
+
+export interface TokenCreateEvent extends Event {
+  data: {
+    operator: string;
+    contract_address: string;
+    data: string;
+  };
+}
+
+export interface TransferEvent extends Event {
+  data: {
+    from: string;
+    to: string;
+    operator: string;
```

Comment:
I don't see "operator" in the emit from ERC20

Reply:
This field should be removed from the interface

Reply:
For now, going to send `GET /abis/<AbiId>/<ContractAdd>/<method>?fly-transaction=<transactionHash>&stream=<streamId>` to eth-connect and fetch the operator value